src/modules/theme/constants/colors.js

```diff
@@ -17,6 +18,7 @@ export const brightColors: ColorsType = {
   textColor: '#000000',
   textSecondaryColor: '#585858',
   textDecorationColor: '#c7c7c7',
+  textDisabledColor: '#dddddd',
```

Comment:
Really hard to read in the header imo, maybe it should be a little bit darker (also the contrast to the textColor is quite good). Maybe #d0d0d0?

Reply:
done